src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 100;
+    private static final int defaultRadius = 10;
```

Comment:
```suggestion
    private static final int MAX_BOUND = 100;
    private static final int DEFAULT_RADIUS = 10;
    private static final String DEFAULT_COLOR = Color.WHITE.name();
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();
```
constants should have names in uppercase
lets also make Random and ColorSupplier global - to use just 1 instance of each class